openedx/core/djangoapps/catalog/utils.py

```diff
@@ -109,15 +109,17 @@ def get_program_types(name=None):
         return []
 
 
-def get_programs_with_type(types=None):
+def get_programs_with_type(types=None, include_hidden=True):
```

Comment:
Product is aware of this. These programs will only be associated with hidden course runs. See https://openedx.atlassian.net/browse/LEARNER-1140.